source/content/guides/build-tools/07-tests.md

```diff
@@ -1,20 +1,11 @@
 ---
 title: Build Tools
 subtitle: Write a New Test
-description: In step six of the Build Tools guide, learn how to use the pre-conifgured site tests, or customize your own.
+description: In step seven of the Build Tools guide, learn how to use the pre-conifgured site tests, or customize your own.
```

Comment:
Should we also mention in here the visual regression tests that were added? Or do they belong in a separate section?

Reply:
Maybe here? 🤔

Reply:
Mentioned in testing with 11cc47a, thank you!